kamon-logback-stackdriver/src/main/scala/kamon/stackdriver/StackdriverMarker.scala

```diff
@@ -0,0 +1,80 @@
+package kamon.stackdriver
+
+import java.util
+
+import kamon.stackdriver.StackdriverMarker._
+import org.slf4j.Marker
+
+import scala.collection.concurrent.TrieMap
+import scala.collection.JavaConverters._
+
+abstract class StackdriverMarker(implicit enc: LogValue.Writer) extends Marker {
+  private val markers = TrieMap.empty[String, Marker]
+
+  def values: Map[String, LogValue] =
```

Comment:
This looks like unneeded, maybe it should be hidden/removed

Reply:
Yeah we should probably remove it, it was from the original suggestion